client/src/containers/projects/custom-project/header/parameters/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  COST_TYPE_SELECTOR,
+  PROJECT_PRICE_TYPE,
+} from "@shared/entities/projects.entity";
+import { z } from "zod";
+
+import { FILTER_KEYS } from "@/app/(overview)/constants";
+import { filtersSchema } from "@/app/(overview)/url-store";
+
+import { INITIAL_COST_RANGE } from "@/containers/overview/filters/constants";
+import { useGlobalFilters } from "@/containers/projects/url-store";
+
+import { Label } from "@/components/ui/label";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+export const PROJECT_PARAMETERS = [
+  {
+    key: FILTER_KEYS[3],
+    label: "Project size",
+    className: "w-[125px]",
+    options: [
+      {
+        label: COST_TYPE_SELECTOR.NPV,
+        value: COST_TYPE_SELECTOR.NPV,
+      },
+      {
+        label: COST_TYPE_SELECTOR.TOTAL,
+        value: COST_TYPE_SELECTOR.TOTAL,
+      },
+    ],
+  },
+  {
+    key: FILTER_KEYS[2],
+    label: "Carbon pricing type",
+    className: "w-[195px]",
+    options: [
+      {
+        label: PROJECT_PRICE_TYPE.MARKET_PRICE,
+        value: PROJECT_PRICE_TYPE.MARKET_PRICE,
+      },
+      {
+        label: PROJECT_PRICE_TYPE.OPEN_BREAK_EVEN_PRICE,
+        value: PROJECT_PRICE_TYPE.OPEN_BREAK_EVEN_PRICE,
+      },
+    ],
+  },
+] as const;
+
+export default function CustomProjectParameters() {
+  const [filters, setFilters] = useGlobalFilters();
```

Comment:
Use specific filters for this page. Global ones only affect only the `/` page, perhaps the name is a bit missleading.